mods_cocina_mappings/mods_to_cocina_abstract.txt

```diff
@@ -0,0 +1,70 @@
+1. Single abstract
+<abstract>This is an abstract.</abstract>
+{
+  "note": [
+    {
+      "value": "This is an abstract.",
+      "type": "summary"
+    }
+  ]
+}
+
+2. Multilingual abstract
+<abstract lang="eng" script="Latn" altRepGroup="1">This is an abstract</abstract>
+<abstract lang="rus" script="Cyrl" altRepGroup="1">Это аннотация.</abstract>
+{
+  "note": [
+    {
+      "parallelValue": [

```

Comment:
Is there a reason we need parallelValue?  Could this be:
```
"note": [

        {
          "value": "This is an abstract.",
          "type": "summary",
          "language": [
            {
              "code": "eng",
              "source": {
                "code": "iso639-2b"
              },
              "script": {
                "code": "Latn",
                "source": {
                  "code": "iso15924"
                }
              }
            }
          ]
        },
        {
          "value": "Это аннотация.",
          "type": "summary",
          "language": [
            {
              "code": "rus",
              "source": {
                "code": "iso639-2b"
              },
              "script": {
                "code": "Cyrl",
                "source": {
                  "code": "iso15924"
                }
              }
            }
          ]
        }
      ]
```

Reply:
Alternatively, can we extract the "type" value up a level, to show we have many translations of the same concept?
```json
"note": [
  {
    "type": "summary",
    "parallelValue":[
      {"value": "This is an abstract.", "language": {...} },
      {"value": "Это аннотация.", "language": {...}}
    ]
  }
]
```

Reply:
Parallel value is needed because there might be multiple notes, and the ones that give the same information in different languages need to be paired so it's clear which e.g. Russian value goes with which English value, or which transliteration goes with a vernacular script. MODS does this via the altRepGroup attribute. Agreed on moving "type" up a level - I'll change that.

Reply:
👍